src/Converter/convertToTile.js

```diff
@@ -41,12 +41,14 @@ export default {
         const builder = layer.builder;
         const parent = requester;
         const level = (parent !== undefined) ? (parent.level + 1) : 0;
+        layer.segments = layer.segments || 16;
 
         const paramsGeometry = {
             extent,
             level,
-            segment: layer.segments || 16,
+            segment: layer.segments,
```

Comment:
By the previous comment:

```suggestion
            segment: segments,
```